composer/yamls/models/gpt3_125m.yaml

```diff
@@ -1,21 +1,23 @@
 train_dataset:
-  c4:
+  streaming_c4:
+    version: 2
+    remote: s3://mosaicml-internal-dataset-c4/mds/2/
+    local: /tmp/mds-cache/mds-c4/
     split: train
-    num_samples: 5120000 # 10e9 tok ~= 256[bs] * 20000[ba] * 2048[msl] = 5120000[sa] * 2048[msl]
```

Comment:
Confirming that you can remove this value here, but now you have to move this to `max_duration`, otherwise these training runs will be very, very long. Would recommend setting something like `max_duration: 20000ba`

Reply:
@knighton  is it possible to specify max_duration w streaming?

Reply:
We don't currently support `max_duration` or such params, open to discussing offline? cc: @karan6181

Reply:
@mvpatel2000 sorry for the confusion, all I meant was that the **Trainer's max_duration** must now be updated from `1ep` to something like `20000ba` because there is no longer an epoch length.

Also, we will need to update the source code for `StreamingC4` to support `concat`, which I can help with.